### ISSUE TITLE
Deprecation and FITS warnings related to WCS

### ISSUE THREAD

**Issue by pllim** [spacetelescope/stwcs#36]
Saw this in regression test, please fix. @mcara or @nden ?

https://ssb.stsci.edu/steuermann/steuermann.cgi?action=log&name=conda_tests_2017-07-30-10:30:02/nott:conda_testing/dev_tests

```
run_proc: pdkrun --dir --environment_already_set /srv/rt/stsci_python/stwcs 
directory /srv/rt/stsci_python/stwcs
COMMAND : 'pdknose --pdk --with-doctest --doctest-tests test.py' (for file /srv/rt/stsci_python/stwcs/test.py) 2017-07-30 10:37:00.812892
WARNING: VerifyWarning: Card is too long, comment will be truncated. [astropy.io.fits.card]
WARNING: AstropyDeprecationWarning: The use of ``AXISCORR`` for D2IM correction has been deprecated.`~astropy.wcs` will read in files with ``AXISCORR`` but ``to_fits()`` will write out files without it. [astropy.wcs.wcs]
WARNING: FITSFixedWarning: CPERROR1= 0.0 / Maximum error of dgeo correction for axis 1 
keyword looks very much like CPERRja but isn't. [astropy.wcs.wcs]
WARNING: FITSFixedWarning: CPERROR2= 0.0 / Maximum error of dgeo correction for axis 2 
keyword looks very much like CPERRja but isn't. [astropy.wcs.wcs]
..../srv/rt/stsci_python/stwcs/test.py:108: RuntimeWarning: cdelt will be ignored since cd is present
  utils.assert_almost_equal(self.w1.wcs.cdelt, self.w1r.wcs.cdelt)
/srv/rt/stsci_python/stwcs/test.py:115: RuntimeWarning: cdelt will be ignored since cd is present
  utils.assert_almost_equal(self.w4.wcs.cdelt, self.w4r.wcs.cdelt)
```

**Comment by mcara** [comment]
@pllim

**[keyword looks very much like CPERRja but isn't.]** It seems this warning dates from [Jan 23, 2015 commit of astropy v5.6](https://github.com/astropy/astropy/commit/7bac4706bb70c6eb973a528a1a592c63ee0e1a8b#diff-b8cd22c26390b4342179aaf014005603R917) Maybe @nden or @stsci-hack know what is the best approach regarding this: keep warning users or set appropriate switches in ``stwcs`` if we expect this in our files. Same applies to ``AstropyDeprecationWarning: The use of ``AXISCORR`` for D2IM correction has been deprecated. ...`` warning.

**[WARNING: VerifyWarning: Card is too long, comment will be truncated.]** In relation to ``drizzlepac`` this was discussed in https://github.com/spacetelescope/drizzlepac/issues/19 I have nothing to add to what I said there.

**[cdelt will be ignored...]** This probably needs fixing of data files in our tests (assuming new data files do not contain ``CDELT`` - I do not know who put this keyword into old test files.)

**Comment by mcara** [comment]
@pllim

**[keyword looks very much like CPERRja but isn't.]** It seems this warning dates from [Jan 23, 2015 commit of astropy v5.6](https://github.com/astropy/astropy/commit/7bac4706bb70c6eb973a528a1a592c63ee0e1a8b#diff-b8cd22c26390b4342179aaf014005603R917) Maybe @nden or @stsci-hack know what is the best approach regarding this: keep warning users or set appropriate switches in ``stwcs`` if we expect this in our files.

**[AstropyDeprecationWarning: The use of ``AXISCORR`` for D2IM correction...]** This warning dates back from [May28th, 2013 - astropy commit 1099](https://github.com/astropy/astropy/pull/1099) Again, maybe @nden or @stsci-hack know what is the best approach regarding this.

**[WARNING: VerifyWarning: Card is too long, comment will be truncated.]** In relation to ``drizzlepac`` this was discussed in https://github.com/spacetelescope/drizzlepac/issues/19 I have nothing to add to what I said there.

**[cdelt will be ignored...]** This probably needs fixing of data files in our tests (assuming new data files do not contain ``CDELT`` - I do not know who put this keyword into old test files.)

**Comment by nden** [comment]
fixed by #78
